apps/wfo-ui/pages/_app.tsx

```diff
@@ -67,37 +68,44 @@ function CustomApp({ Component, pageProps }: AppProps) {
                     colorMode="light"
                     modify={defaultOrchestratorTheme}
                 >
-                    <Head>
-                        <title>Welcome to example-orchestrator-ui!</title>
-                    </Head>
-                    <main className="app">
-                        <OrchestratorConfigProvider
-                            initialOrchestratorConfig={
-                                initialOrchestratorConfig
-                            }
-                        >
-                            <QueryClientProvider
-                                client={queryClient}
-                                contextSharing={true}
+                    <ApiClientContextProvider
```

Comment:
This seems to be a provider only for Axios (and thus Forms), maybe we should move this provider into the form scope and out of the _app.tsx

Reply:
It has to be on the App level to be able to use the values from 'constants.ts' so I think it needs to be here